src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -203,11 +204,6 @@ public ConfidentialClientApplicationBuilder WithClientAssertion(Func<string> cli
         /// <returns>The builder to chain the .With methods</returns>
         public ConfidentialClientApplicationBuilder WithAzureRegion(string azureRegion = ConfidentialClientApplication.AttemptRegionDiscovery)
         {
```

Comment:
there should be a test that failed? right?

Reply:
did you check that we have a test which passes in null?

Reply:
yes, I was surprized that no test failed, right.
But I'll add one ...